src/main/java/com/twilio/oai/api/PythonApiResourceBuilder.java

```diff
@@ -28,8 +30,19 @@ public ApiResourceBuilder updateOperations(final Resolver<CodegenParameter> code
             if (co.operationId.startsWith("list")) {
                 addOperationName(co, "Page");
             }
+
+            dependents.values().forEach(dependent -> {
+                dependent.setImportName(dependent.getImportName().replace("Instance", ""));
+            });
+
         }
+        return this;
+    }
 
+    @Override
+    public ApiResourceBuilder updateVersionImports(){
+        ArrayList<DirectoryStructureService.DependentResource> versionResources = (ArrayList<DirectoryStructureService.DependentResource>) this.directoryStructureService.getAdditionalProperties().get("versionResources");
+        versionResources.forEach(versionResource -> versionResource.setImportName(versionResource.getImportName().replace("Instance", "")));
```

Comment:
Could use the `resourceName` in the templates instead of altering the import name?

https://github.com/twilio/twilio-oai-generator/blob/475e098f916a6f844ed83b25f8805031bae5d0fa/src/main/java/com/twilio/oai/DirectoryStructureService.java#L163

Reply:
We would need to add "List" to it, so would still need similar logic somewhere. The import name shouldn't have "Instance" in it anyway, so figured that approach would work.

Reply:
I'd prefer to add another property to the DependentResource class (`listName` maybe) to reduce the logic.